src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,100 @@
  */
 
 public class Plane {
```

Comment:
try creating object of Plane (for this purpose create main class and method) and write your code in comment under this

Reply:
What do you mean?